Content.Shared/Cluwne/CluwneComponent.cs

```diff
@@ -24,6 +25,39 @@ public sealed class CluwneComponent : Component
     [DataField("emoteId", customTypeSerializer: typeof(PrototypeIdSerializer<EmoteSoundsPrototype>))]
     public string? EmoteSoundsId = "Cluwne";
 
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float Cluwinification = 0.15f;
+
+    /// <summary>
+    /// Should be true if this is a cluwne.
+    /// </summary>
+    [DataField("isCluwne")]
+    public bool IsCluwne = true;
+
+    /// <summary>
+    /// The autoemote sound to play.
+    /// </summary>
+    [DataField("autoEmoteSound")]
+    public readonly string AutoEmoteSound = "CluwneGiggle";
+
+    /// <summary>
+    /// If this is true then has a % chance to transform target into a cluwne when hit with melee.
+    /// </summary>
+    [DataField("cluwneOnMelee")]
+    public bool CluwneOnMelee = false;
```

Comment:
This just doubles the above field when you could set it to 0 and get the same effect.